packages/ui/components/editor/Editor.tsx

```diff
@@ -84,13 +84,13 @@ export const Editor = (props: TextEditorProps) => {
           />
           <div
             className={classNames("editor-inner scroll-bar", !editable && "!bg-subtle")}
-            style={{ height: props.height }}>
+            style={{ height: props.height, overflow: 'hidden auto' }}>
             <RichTextPlugin
               contentEditable={
                 <ContentEditable
                   data-testid="editor-input"
                   readOnly={!editable}
-                  style={{ height: props.height }}
+                  style={{ maxHeight: '250px' }}
```

Comment:
Hey, don't set any hardcoded values here; the editor is used widely in our app

Reply:
#16231 check this